portfolio/src/main/java/com/google/sps/servlets/UserServlet.java

```diff
@@ -33,18 +33,18 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     UserService userService = UserServiceFactory.getUserService();
     if (userService.isUserLoggedIn()) {
       String userEmail = userService.getCurrentUser().getEmail();
-      String urlToRedirectToAfterUserLogsOut = "/user";
+      String urlToRedirectToAfterUserLogsOut = "/";
       String logoutUrl = userService.createLogoutURL(urlToRedirectToAfterUserLogsOut);
       
       loggedIn = true;
-      response.getWriter().println("<p>Hello you are logged in, " + userEmail + "!</p>");
+      response.getWriter().println("Logged in, " + userEmail + "!");
       response.getWriter().println("<p>Logout <a href=\"" + logoutUrl + "\">here</a>.</p>");
     } else {
-      String urlToRedirectToAfterUserLogsIn = "/user";
+      String urlToRedirectToAfterUserLogsIn = "/";
```

Comment:
Same question here: does this just take user back to home page?

Reply:
Yes